packages/node_modules/pouchdb-utils/src/cloneBinaryObject-browser.js

```diff
@@ -14,14 +14,8 @@ function cloneBinaryObject(object) {
   if (object instanceof ArrayBuffer) {
     return cloneArrayBuffer(object);
   }
```

Comment:
```suggestion
  if (object instanceof ArrayBuffer) {
    return object.slice();
  }
```
Since we dropped support for IE, `cloneArrayBuffer` can be removed too 😉